apisix/discovery/nacos/schema.lua

```diff
@@ -54,6 +54,49 @@ return {
         },
         access_key = {type = 'string', default = ''},
         secret_key = {type = 'string', default = ''},
+        others = {
```

Comment:
what does this `others` mean?

This looks like a very big PR now. Please explain this PR as a proposal in #10799, we need to have a consensus on "HOW" to implement this feature.

If we continue to review code without agreeing on "HOW" there will be a lot of back and forth during reviews and a lot of time wasted.

Reply:
The 'others' field represents a list of additional Nacos configurations. Since the previous schema for Nacos configuration was an object, directly changing it to an array would render existing configurations incompatible. Therefore, I've utilized the 'others' field to extend this functionality. Of course, if we decide that backward compatibility is not necessary, we can redesign this configuration schema. Currently, my implementation involves adding a new layer of 'nacosName' within the original 'applications' map. This layer corresponds to different Nacos configurations. Additionally, the asynchronous tasks for fetching Nacos configurations have been modified from fetching from a single Nacos instance to fetching from multiple instances. They are organized into different maps based on the 'name' specified in the configuration.